doc/languages-frameworks/python.section.md

```diff
@@ -538,6 +538,95 @@ buildPythonPackage rec {
 ```
 Note also the line `doCheck = false;`, we explicitly disabled running the test-suite.
 
+#### Testing Python Packages
+
+It is highly encouraged to have testing as part of the package build. This
+helps to avoid situations where the package was able to build and install,
+but is not usable at runtime. Currently, all packages will use the `test`
+command provided by the setup.py. However, this is currently deprecated
```

Comment:
```suggestion
command provided by the setup.py (i.e. ``python setup.py test``). However, this is currently deprecated
```
Not sure if it's worth being explicit here.

Reply:
I was thinking about this as well, but the nix-correct command would be `${python.interpreter} setup.py test` since we also support `pypy`. However, it's just an example.. and might be more relatable if just `python` is used

Reply:
meh, I like this better